patterns/tombstones.md

```diff
@@ -39,3 +39,5 @@ We can encourage deletion across the network quite well by using Tombstones.
 - Depending on the protocol or library you choose for storage, it may not be possible to delete historical data. In this case, tombstones only 'hide' data from view, rather than delete it from disk.
 
 ## References & Where to Learn More
+
+"Network heartbeats" can track which peers with a copy of data continue to share it after a tombstone has been sent. See [network health indicators](patterns/network-health-indicator.md) for more on this approach.
```

Comment:
This is awesome. We never really define what a 'heartbeat' is but as I understand it's a pretty well-known strategy used for networking protocols. I'd like to expand on it a bit more, by perhaps linking somewhere that defines it.

Reply:
Whoops, you're absolutely right. I'm coming from a network protocol background and the terminology slipped in. "Heartbeat" in this context is in the loosest sense "some kind of regular message indicating continued availability of a resource", which is effectively what we've described in 'network health indicators'. The wikipedia article you've linked to is great.